### PR TITLE
#162727458 Setup testing Environment

### PR DESCRIPTION
**Pull request by wombolo**
#### What does this PR do?
Setup testing Environment using Mocha & Chai

#### Description of Task to be completed?
Ensure that all backend code is working as intended

    
#### How should this be manually tested?
On the root folder of the project in the terminal,
Run `npm install`
Then `npm run test`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162727458

### CODE REVIEW

**Review by daniellamarr — APPROVED** [review]
Looks good.

File: test/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+// import app from '../index.js';
```
Comment (andela-moe) [inline test/index.spec.js]: If this is not in use I believe we should not comment it out but delete it. Please resolve.

**Review by andela-moe — COMMENTED** [review]
Please review comments.

File: test/index.spec.js (hunk 1 above)
Comment (wombolo) [reply test/index.spec.js]: Un-needed comment has been removed

**Comment by andela-moe** [comment]
You have conflicts you will need to resolve before this PR can be merged.

**Pull request merged**